### ISSUE TITLE
question about dropout in the implementation

### ISSUE THREAD

**Issue by lai-agent-m** [lukemelas/EfficientNet-PyTorch#162]
drop_connect_rate *= float(idx) / len(self._blocks)
why you linearly increase the learning rate as the depth of the network increases? I don't find this detail on the paper, is this trick on the original code??
and why you name it drop_connect_rate?  I feel that it is simply dropout rate. you didn't apply the dropout on the weight

**Comment by lukemelas** [comment]
Yup, it's in the original code: https://github.com/tensorflow/tpu/blob/master/models/official/efficientnet/efficientnet_model.py#L688-L689
It's not the dropout rate -- it's the dropconnect rate (they're different).

**Comment by lai-agent-m** [comment]
DropConnect instead sets a randomly selected subset of weights within the network to zero. 
I may misunderstood something, can you just show me some evidence that the dropout is applied on the weights. From what I saw, the dropout is applied to feature map.
